hive-bigquery-parent/pom.xml

```diff
@@ -56,77 +56,147 @@
 
     <properties>
         <revision>2.0.0-SNAPSHOT</revision>
-        <nexus.remote.skip>false</nexus.remote.skip>
 
+        <!-- Dependencies -->
         <avro.version>1.11.1</avro.version>
-        <jackson.version>2.12.7.1</jackson.version>
-        <protobuf.version>3.21.12</protobuf.version>
-        <legacy-bigquery-connector.version>hadoop2-1.0.0</legacy-bigquery-connector.version>
         <bigquery-connector-common.version>0.31.0</bigquery-connector-common.version>
+        <gcs-connector.version>hadoop2-2.2.14</gcs-connector.version>
         <google-cloud-dataproc.version>4.7.0</google-cloud-dataproc.version>
         <google-cloud-storage.version>2.20.1</google-cloud-storage.version>
         <grpc.version>1.53.0</grpc.version>
-        <junit.version>5.9.2</junit.version>
+        <guava.version>31.1-jre</guava.version>
         <hamcrest.version>2.2</hamcrest.version>
         <hiverunner.version>6.1.0</hiverunner.version>
-        <truth.version>1.1.3</truth.version>
```

Comment:
Why to remove Google Truth?

Reply:
See above comment. I'll resolve this one for now.